src/Microsoft.AspNetCore.Mvc.Razor/Internal/CompositeTagHelperDescriptorResolver.cs

```diff
@@ -10,22 +10,18 @@ namespace Microsoft.AspNetCore.Mvc.Razor.Internal
     public class CompositeTagHelperDescriptorResolver : ITagHelperDescriptorResolver
```

Comment:
I don't think so, but it's my understanding that we'll be adding the ViewComponentTagHelperDescriptorResolver back in, so I figured we didn't want to do too much work rearchitecting this in the meantime.